### DISCUSSION TITLE
Purpose of "mode_base" in  config

### DISCUSSION THREAD

**Discussion by SodaSyrup** [seerge/g-helper#2875] [Q&A] [answered]
Hello, i wanted to know what difference it makes using different base modes for custom profiles. I read through some other discussions but there wasnt really a clear cut answer or i couldnt find it. So what exactly happens when the base mode is silent vs balanced vs turbo when making a custom profile?

**Comment by seerge** [comment]
@SodaSyrup each base mode is a "preset" stored in BIOS, consisting with lots of different params :) Some of them (like power limits and fan curves can be altered).

**Reply by SodaSyrup** [reply]
is it huge stuff thats very obvious? or just some minor things here and there where its enough to say "when making a performance profile just use turbo and when doing an eco profile just use silent preset"?

**Reply by seerge** [reply, marked as answer]
@SodaSyrup it's usually GPU power limits (that you can't change unless you have 2024 model), CPU temp limit is different for Silent mode usually, some other things, probably some internal stuff as well ....

In Armoury when you make "Manual" mode it always takes Turbo as base + customizations. Here you can take any :) 

I would say use base profile that as close to your needs as possible, and customize only things that you really want to change (in most of the cases defaults are fine too)

**Reply by SodaSyrup** [reply]
i made a performance profile for myself and looked at power user settings and saw the "mode_base" was set to 0 (which i think is balanced?) so i put it to 1. just wanted to make sure if there is anything at all. thank you for your quick and helpful reply!

**Reply by seerge** [reply]
@SodaSyrup when you click (+) it uses currently active mode as base. You can change it in config of course too.

You can also just customize default modes, without copying them before. Effect will be same :)

**Reply by SodaSyrup** [reply]
i wanted to keep the defaults for now, doing some very basic testing and i use them as reference and all that. but thats good to know too, thank you!
